### DISCUSSION TITLE
install/commercial/start-install/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1298] [Announcements]
# install/commercial/start-install/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/commercial/start-install/

**Comment by panpan0000** [comment]
离线物料：发现并不只下载镜像，跳过ISO和os-package，可能需要文档注明下

**Reply by panpan0000** [reply]
离线物料：发现并不能只下载镜像，然后跳过ISO和os-package（不下载），会报错
8:48AM: Pushing files in offline-iso to local minio.. ⏫
....
8:48AM: empty ISO_IMG_FILE

可能需要文档注明下

**Reply by jzhupup** [reply]
ok

**Comment by KubeKyrie** [comment]
注：一键下载所需离线包不包含ISO 操作系统镜像文件

**Comment by gssjl2008** [comment]
ubuntu jammy: 安装，一共遇到的几个问题。 
1. https://docs.daocloud.io/install/commercial/start-install/#ospackage  这个文档，没有提供 ubuntu jammy的安装包，使用 kubean社区的os-pkgs-ubuntu2204-v0.16.3.tar.gz 这个离线包中，没有提供ipset包，V0.17.3的包中有提供。 
2.  ubuntu jammy 需要确保 systemd-resolved  这个服务 enable， 否则 container的容器，无法挂载 /var/run/systemd/resolve/ 目录
3. kpanda-controller  启动失败，需要设置： fs.inotify.max_user_instances  和 fs.inotify.max_user_watches 这两个内核参数。

**Comment by carlo-zhang** [comment]
osPackage 离线包 不是最新版的
